org.eclipse.lsp4j/src/main/java/org/eclipse/lsp4j/services/NotebookDocumentService.java

```diff
@@ -24,7 +23,7 @@ public interface NotebookDocumentService {
 	/**
 	 * The open notification is sent from the client to the server when a notebook document
 	 * is opened. It is only sent by a client if the server requested the synchronization
-	 * mode {@code notebook} in its {@link ServerCapabilities#notebookDocumentSync} capability.
+	 * mode {@code notebook} in its {@code ServerCapabilities.notebookDocumentSync} capability.
```

Comment:
Not sure whether it'd worth the effort, but potentially we could use constructs like this in such cases:

```suggestion
	 * mode {@code notebook} in its {@link ServerCapabilities#getNotebookDocumentSync() notebookDocumentSync} capability.
```

Reply:
That is a good solution. I will do that. I hadn't wanted `{@link ServerCapabilities#getNotebookDocumentSync()}` alone, because it looked "bad". Bug something like you have, although probably as below instead, I think is ok.
```suggestion
	 * mode {@code notebook} in its {@link ServerCapabilities#getNotebookDocumentSync() ServerCapabilities.notebookDocumentSync} capability.
```